szperacz/logger.py

```diff
@@ -1,18 +1,36 @@
 
 
 class Logger:
+    global log_level
+    log_level = 0
+    level = log_level
```

Comment:
1. Move initialization of `log_level` to constructor
2. No need for `level` variable. Please remove